iothub/service/src/AmqpServiceClient.cs

```diff
@@ -389,6 +389,7 @@ private Task<SendingAmqpLink> CreateSendingLinkAsync(TimeSpan timeout)
         /// <inheritdoc/>
         protected override void Dispose(bool disposing)
         {
+            base.Dispose(disposing);
```

Comment:
q - what operation is associated with this base.dispose() call?

Reply:
There is nothing, the pattern should exist though regardless of what happens in the base class.